src/Component.ts

```diff
@@ -60,7 +65,7 @@ type RestParam<T extends EventMap, K extends EventKey<T>>
  * @ko 컴포넌트의 이벤트을 관리할 수 있게 하는 클래스
  * @alias eg.Component
  */
-class Component<T extends EventMap> {
+class Component<T extends EventMap = {}> {
```

Comment:
@WoodNeck 

{} means there are no events.

With EventMap, any event can be used.

Reply:
That enables making a Component with no events, like
```ts
class SomeClass extends Component { }
```

> @WoodNeck
> Use default type.

I thought the default type you said was that feature. How should I change it?

Reply:
@WoodNeck 

I think that there will be no errors for existing users.

```ts
class Component<T extends EventMap =  EventMap>
``